pkg/ui/workspaces/cluster-ui/src/insights/workloadInsightDetails/transactionInsightDetailsStmtsTab.tsx

```diff
@@ -47,13 +51,15 @@ const stmtColumns: ColumnDescriptor<StatementInsightEvent>[] = [
   {
     name: "startTime",
     title: "Start Time",
-    cell: (item: StatementInsightEvent) => item.startTime.format(DATE_FORMAT),
+    cell: (item: StatementInsightEvent) =>
+      item.startTime.format(DATE_WITH_SECONDS_AND_MILLISECONDS_FORMAT_24_UTC),
```

Comment:
can you update the column title to have `Star Time (UTC)`, and then the value won't need the `UTC`